posts/2024-05-05-liberty-project-starter-guide-IntelliJ.adoc

```diff
@@ -0,0 +1,209 @@
+            <artifactId>liberty-maven-plugin</artifactId>
+            <version>3.10.2</version>
+        </plugin>
+    </plugins>
+</pluginManagement>
+----
+
+//explain lmp here
+In this example, the `liberty-maven-plugin` from `io.openliberty.tools` has been introduced as a plugin for this project. The Liberty Maven Plugin offers several goals for managing a Liberty runtime, including tasks such as downloading and installing the Liberty runtime, starting or stopping a Liberty server in development mode, installing features, and deploying applications. Please visit link:https://github.com/OpenLiberty/ci.maven/blob/main/README.md[Liberty Maven Plugin Document] for more information.
+
+
+[#libertyToolsWithDevMode]
+== Rapid application development with Liberty Tools
+
+=== Introducing Liberty Tools
+Liberty Tools enhances the application development experience with Open Liberty by providing convenient features. It includes a Liberty Dashboard for organizing projects and integrates Liberty dev mode directly into your IDE. 
+
+The Liberty Dashboard effectively manages Maven projects, seamlessly integrating configurations for Open Liberty. Moreover, it facilitates rapid development of MicroProfile and Jakarta EE applications by offering automatic code blocks, auto-complete functionality, and real-time syntax validation. With just a few clicks, you can start or stop your app, run tests, and check reports. 
+
+Furthermore, Liberty dev mode can swiftly apply code changes to your running app without needing to restart the server, ensuring faster development.
+
+You may opt to use Liberty Tools for enhanced convenience:
+
+image::img/blog/Liberty-Tools-Example.png[Liberty Tools Example image, title="An example integrating Liberty Dashboard from Liberty Tools into a Maven project with IntelliJ IDEA", width=30%,align="center"]
+
+Liberty Tools offer three methods to assist you in starting your Liberty application in development mode: Start, Start with configuration, or Start in a container. To start your application in dev mode, click the `Start` action for your application in the Liberty Dashboard.
+
+To effectively use the Liberty Tools for Liberty dev mode and other Liberty dashboard option, consulting the link:https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/docs/user-guide.md#run-your-application-on-liberty-using-dev-mode[user guide provided with IntelliJ IDEA] is recommended. For more information about liberty development mode and container support for development mode, please visit the link:https://openliberty.io/docs/latest/development-mode.html[dev mode document].
+
+You can also visit articles on IBM Developers for More information about Liberty Tools: link:https://developer.ibm.com/articles/awb-effective-cloud-native-development-open-liberty-intellij-idea/[Effective cloud-native Java app development with Open Liberty in IntelliJ IDEA]
+
+Additionally, you can use the command line.
```

Comment:
```suggestion
```
Although you can use dev mode on the cmd line, I think we should stick with the IDE here.